app/src/main/java/com/puzzlebench/clean_marvel_kotlin/domain/model/RealmCharacter.kt

```diff
@@ -1,12 +1,13 @@
 package com.puzzlebench.clean_marvel_kotlin.domain.model
 
-import com.puzzlebench.clean_marvel_kotlin.CONSTS
+import com.puzzlebench.clean_marvel_kotlin.EMPTY_VALUE
+import com.puzzlebench.clean_marvel_kotlin.ZERO_VALUE
 import io.realm.RealmObject
 import io.realm.annotations.PrimaryKey
 
 open class RealmCharacter(
-        @PrimaryKey var id: Int = CONSTS.ZERO_VALUE,
-        var name: String = CONSTS.EMPTY_VALUE,
-        var description: String = CONSTS.EMPTY_VALUE,
+        @PrimaryKey var id: Int = ZERO_VALUE,
+        var name: String = EMPTY_VALUE,
```

Comment:
you could use de constatnts file you create

Reply:
Done

Reply:
my bad